src/node.cc

```diff
@@ -3320,6 +3323,8 @@ static void PrintHelp() {
          "  --force-fips          force FIPS crypto (cannot be disabled)\n"
 #endif  /* NODE_FIPS_MODE */
 #endif /* HAVE_OPENSSL */
+         "  --openssl-conf-system use the file system kernel configuration "
```

Comment:
Perhaps, `openssl-system-conf`? Sounds a bit more logical to me. (Though, I'm not a native speaker myself, so I could be wrong here too).